### ISSUE TITLE
question about reproduction

### ISSUE THREAD

**Issue by yk907** [ShiqiYu/OpenGait#252]
Hello, due to the limitation of computing resources, I have to reduce batchsize [8,8] to [2,8] (biggait). If I want to reproduce as much as possible, how should I adjust batchsize or hyperparameter, such as reducing the learning rate? Thank you for your help.

**Comment by Bugjudger** [comment]
You can try reducing frames_num_fixed from 30 to 8, or reducing sils_size from 32 to 20, or changing the dim of x_norm patchttokens_mid4 from concat 4x384 to mean 384. But all these operations will decrease the results. We haven't explored this yet.

**Comment by yk907** [comment]
Thank you for your soon reply, it work.
